test/regression/road_curve/spiral_ground_curve_test.cc

```diff
@@ -0,0 +1,610 @@
+  EXPECT_NEAR(BruteForcePathLengthIntegral(dut_, kP0, kMidP, kOrder), dut_.ArcLength() / 2.,
+              kToleranceForBruteForceIntegral);
+  EXPECT_NEAR(BruteForcePathLengthIntegral(dut_, kP0, kP1, kOrder), dut_.ArcLength(), kToleranceForBruteForceIntegral);
+}
+
+// Tests the case in which the curvature derivative is not normalized.
+class DenormalizedSpiralGroundCurveTest : public BaseSpiralGroundCurveTest {
+ protected:
+  const Vector2 kXy0{0., 0.};
+  static constexpr double kStartHeading{0.};
+  static constexpr double kStartCurvature{0.};
+  static constexpr double kEndCurvature{0.2};
+  static constexpr double kArcLength{10.};
+  static constexpr double kP0{0.};
+  static constexpr double kMidP{5.};
+  static constexpr double kP1{10.};
+  const SpiralGroundCurve dut_{kLinearTolerance, kXy0,       kStartHeading, kStartCurvature,
+                               kEndCurvature,    kArcLength, kP0,           kP1};
+};
+
+TEST_F(DenormalizedSpiralGroundCurveTest, G) {
+  EXPECT_TRUE(AssertCompare(CompareVectors({0., 0.}, dut_.G(kP0), kLinearTolerance)));
+  EXPECT_TRUE(AssertCompare(CompareVectors({4.9681738083, 0.4227178689}, dut_.G(kMidP), kLinearTolerance)));
+  EXPECT_TRUE(AssertCompare(CompareVectors({9.0253069245, 3.1602035564}, dut_.G(kP1), kLinearTolerance)));
+}
+
+TEST_F(DenormalizedSpiralGroundCurveTest, GDot) {
+  EXPECT_TRUE(AssertCompare(CompareVectors({1., 0.}, dut_.GDot(kP0), kLinearTolerance)));
+  EXPECT_TRUE(AssertCompare(CompareVectors({0.9999987335, 0.0015915488}, dut_.GDot(kMidP), kLinearTolerance)));
+  EXPECT_TRUE(AssertCompare(CompareVectors({0.9999797358, 0.0063661547}, dut_.GDot(kP1), kLinearTolerance)));
+}
+
+TEST_F(DenormalizedSpiralGroundCurveTest, Heading) {
+  EXPECT_NEAR(0., dut_.Heading(kP0), kLinearTolerance);
+  EXPECT_NEAR(0.0015915494309189536, dut_.Heading(kMidP), kLinearTolerance);
+  EXPECT_NEAR(0.006366197723675814, dut_.Heading(kP1), kLinearTolerance);
+}
+
+TEST_F(DenormalizedSpiralGroundCurveTest, HeadingDot) {
+  EXPECT_NEAR(0., dut_.HeadingDot(kP0), kLinearTolerance);
+  EXPECT_NEAR(0.007978845608028654, dut_.HeadingDot(kMidP), kLinearTolerance);
+  EXPECT_NEAR(0.015957691216057307, dut_.HeadingDot(kP1), kLinearTolerance);
+}
+
+TEST_F(DenormalizedSpiralGroundCurveTest, GInverse) {
+  EXPECT_NEAR(kP0, dut_.GInverse({0., 0.}), kGInverseTolerance);
+  EXPECT_NEAR(kMidP, dut_.GInverse({4.9681738083, 0.4227178689}), kGInverseTolerance);
+  EXPECT_NEAR(kP1, dut_.GInverse({9.0253069245, 3.1602035564}), kGInverseTolerance);
+}
+
+TEST_F(DenormalizedSpiralGroundCurveTest, ProperNormalizationYieldsArcLengthToBekP1MinuskP0) {
+  EXPECT_NEAR(BruteForcePathLengthIntegral(dut_, kP0, kMidP, kOrder), dut_.ArcLength() / 2.,
+              kToleranceForBruteForceIntegral);
+  EXPECT_NEAR(BruteForcePathLengthIntegral(dut_, kP0, kP1, kOrder), dut_.ArcLength(), kToleranceForBruteForceIntegral);
+}
+
+}  // namespace
+}  // namespace test
+}  // namespace road_curve
+}  // namespace malidrive
```

Comment:
nit: add empty line

Reply:
Solved in [6bdfb63](https://github.com/maliput/maliput_malidrive/pull/276/commits/6bdfb63d566ee3c22492764a10a4973c80fadafd)